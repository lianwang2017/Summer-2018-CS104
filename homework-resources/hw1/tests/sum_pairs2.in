9
184
650
8
509
73
997
894
460
97
