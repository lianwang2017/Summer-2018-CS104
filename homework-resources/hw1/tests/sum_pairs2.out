281
1110
902
1506
146
